core/controllers/base.py

```diff
@@ -300,21 +300,7 @@ def render_downloadable_file(self, values, filename, content_type):
             'attachment; filename=%s' % filename)
```

Comment:
Shouldn't there be changes to LogoutPage as well?

Reply:
Done.